jwala-common/src/main/java/com/cerner/jwala/common/rule/group/GroupNameRule.java

```diff
@@ -16,8 +17,43 @@ protected MessageResponseStatus getMessageResponseStatus() {
         return FaultType.INVALID_GROUP_NAME;
     }
 
+
+    protected MessageResponseStatus getMessageResponseGroupLengthStatus() { return FaultType.GROUP_NAME_TOO_LONG; }
```

Comment:
Hi sure, I will rename it to something shorter, I did it because it was done similarly in invalid group name.